internal/app/secretless/proxy.go

```diff
@@ -130,10 +131,20 @@ func (p *Proxy) Run() {
 	for msg := range p.runEventChan {
 		switch msg {
 		case START, RESTART:
+			// Set the health check to show that we are ready
+			if msg == START {
+				util.SetAppInitializedFlag()
+			}
+
+			// Mark app as "live"
+			util.SetAppIsLive(false)
+
 			p.cleanUpListeners()
 			p.Listeners = make([]plugin_v1.Listener, 0)
 
 			// TODO: Delegate logic of this `if` check to connection managers
+			// XXX: App is not marked as live if we are not listening to anything. This
```

Comment:
Instead of having this as a comment here (or at least, in addition to) - why not have this as part of the documentation right now?

> If Secretless is not provided with a configuration (eg it is not listening to anything), the `live` endpoint will return 503  as the broker does not have any active listeners.